src/main/java/net/mcreator/ui/modgui/ProcedureGUI.java

```diff
@@ -274,7 +274,7 @@ public Component getListCellRendererComponent(JList<? extends Dependency> list,
 			setForeground(isSelected ? (Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR") : col.brighter());
 			ComponentUtils.deriveFont(this, 14);
 			setText(value.getName());
-			setToolTipText(value.getName() + ", type: " + value.getRawType());
+			setToolTipText(value.toString());

```

Comment:
The changes related to the dependency tooltip being localizable could be moved to a smaller PR

Reply:
Should I move this single string to that PR? Because the localizable dependency data provided by `Dependency#toString()` is used in JS bridge here so I can't really revert it entirely inside this PR.

Reply:
The `toString` here, the method override in `Dependency` and the localization line could be moved to the other PR. Then you can merge master branch into this one after the other PR is merged

Reply:
Not sure to understand how that little changes make PR less atomic, though 😟
#3911

Reply:
Changes unrelated to the PR make it less atomic. This localization is not directly related to the purpose of this PR

Reply:
✔️ Fixed